src/gwt/src/org/rstudio/studio/client/workbench/views/help/HelpPane.java

```diff
@@ -176,19 +176,6 @@ else if (scrollPos_ != null)
 
       prefs_.helpFontSizePoints().bind((Double value) -> refresh());
       
-      events_.addHandler(MouseNavigateEvent.TYPE, (MouseNavigateEvent event) ->
```

Comment:
Will help pane navigation still work for RStudio Server with this code removed? The other code you added is desktop-only (DesktopApplicationHeader).

Reply:
Yes; it appears that event is only fired on Desktop:

https://github.com/rstudio/rstudio/blob/fbb7fafacf1e185ea2d782b0ef35ded9f613a54e/src/gwt/src/org/rstudio/studio/client/application/DesktopHooks.java#L258-L261

https://github.com/rstudio/rstudio/blob/fbb7fafacf1e185ea2d782b0ef35ded9f613a54e/src/gwt/src/org/rstudio/studio/client/workbench/views/source/Source.java#L346-L347

(and as added in this PR). Perhaps we should rename the event to make it clear it's intended to be desktop-only?